assets/src/js/composer.js

```diff
@@ -784,15 +784,15 @@
                     revert:            true,
                     connectToSortable: '.page-composer__container__children',
                     accept: function (source) {
-                        var blockWhitelist = $(this).attr('data-block-whitelist');
-                        if (blockWhitelist === '') {
+                        var blockAllowlist = $(this).attr('data-block-allowlist');
```

Comment:
Technically speaking, this is a BC break.

I think we could risk this, unless you leave an upgrade note.

Reply:
I'm new to this, what is the best way to leave an upgrade note?

Reply:
There is a UPGRADE.md file, you could add an entry to this file to explain the change.

Another solution would be to select
´´´
This.data(whitelist) || this.data(allowlist)
´´´
With a next major to remove the whitelist part